### DISCUSSION TITLE
HELP！

### DISCUSSION THREAD

**Discussion by XiaoyiGao** [zeek/zeek#3457] [Help] [answered]
I just want to know how could I get Zeek in Win11！I really need help！Please！

**Comment by timwoj** [comment, marked as answer]
This isn't the proper venue for questions like this. Please see the documentation for how to build Zeek on Windows in the section on Windows here: https://docs.zeek.org/en/master/install.html#required-dependencies. If you have other questions about how to build or run it, feel free to join the [Zeek Slack](https://zeek.org/slack server) which is where we tend to handle support questions.
